documentation/reference/search-definitions-reference.html

```diff
@@ -2604,8 +2604,8 @@ <h2 id="field-types">Field types</h2>
 }
 field identities type map&lt;string, person&gt; {
     indexing: summary
-    struct-field key { indexing: attribute }
-    struct-field value.last_name { indexing: attribute }
```

Comment:
This is not a typo. See https://github.com/vespa-engine/system-test/blob/master/tests/search/struct_and_map_types/attribute_fields/test.sd#L32 for similar example.

Reply:
you are of course right. the problem with the examples is the inverted order - it should start with primitive types first, and then add complications like structs (I confused myself reading this). @jobergum har asked for a separate guide for multivalue fields - I think we could stuff searchdef examples there and link this this from the reference instead